src/screens/AdminPortal/FundCampaignPledge/FundCampaignPledge.tsx

```
@@ -242,245 +241,247 @@ const fundCampaignPledge = (): JSX.Element => {
+          <div className={styles.titleContainer}>
+            <h3>{campaignInfo?.name}</h3>
+            <span>
+              {t('pledges.endsOn')}{' '}
+              {dayjs(campaignInfo?.endDate).format('DD/MM/YYYY')}
+            </span>
+          </div>
+          <div className={styles.progressContainer}>
+            <div className="d-flex justify-content-center">
+              <fieldset
+                className={`btn-group ${styles.toggleGroup}`}
+                role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)